tgui/packages/tgui/interfaces/Vote.js

```diff
@@ -26,6 +26,9 @@ export const Vote = (props, context) => {
           )}
           <Section title="Start Voting">
             <StartVoteOptions />
+          </Section>
+		  <Section title="Description">
+			<p>{question}</p>
```

Comment:
There is no point to use `<p>` here because question is the only component in this `<Section>`. Also what happened to indentations here? Use https://marketplace.visualstudio.com/items?itemName=Goonstation.goonstation-extpack as it contains all needed extensions for Byond + ESLint and few other things

Reply:
fixed